client/components/CovDateFilter.vue

```diff
@@ -9,7 +9,9 @@
           {{ errorMessage }}
         </div>
       </transition>
-      <cov-button class="cov-button--filter" label="Limpar filtro" @click="clearFilter" />
+      <transition name="button">
```

Comment:
mude o nome para o nome da animação, button não faz sentido, não existe uma animação de button, mas existe um fade por exemplo

Reply:
bom se for o caso, vou deixar a mesma transição do `error`.